test/attr/Inputs/DeprecatedSymbolsModule.swift

```diff
@@ -0,0 +1,20 @@
+@available(*, deprecated)
+func importedDeprecatedFunc() {}
+
+@available(*, deprecated)
+func importedDeprecatedFuncWithReturn() -> String {
+    return "Donda"
+}
+
+@available(*, deprecated)
+var importedDeprecatedVar = "910210"
```

Comment:
Should there be a use of this in the test file?

Reply:
There, used it